test_error_reporting.py

```
@@ -0,0 +1,322 @@
+
+
+async def test_error_with_context(error_reporter):
+    """컨텍스트 정보가 포함된 에러 테스트"""
+    print("\n" + "=" * 60)
+    print("3️⃣  테스트 3: 컨텍스트 정보가 포함된 에러")
+    print("=" * 60)
+
+    try:
+        # Simulate an API error
+        data = {"user_id": 12345, "action": "buy", "symbol": "BTC"}
+        price = data["price"]  # KeyError
```

Ruff (0.14.3)

Local variable `price` is assigned to but never used

Remove assignment to unused variable `price`

(F841)